scripts/globals/effects/naturalists_roll.lua

```diff
@@ -0,0 +1,26 @@
+-----------------------------------
+--
+--
+--
+-----------------------------------
+
+-----------------------------------
+-- onEffectGain Action
```

Comment:
Request: don't add the little function headers. I recently mass removed them from the project because all they do is repeat the function name.

Reply:
File headers good, function headers bad :+1:

Reply:
I can address that in my next PR. Since they were present in the existing cor roll effects I kept them in for consistency sake.